.github/CODEOWNERS

```diff
@@ -8,75 +8,75 @@
 # Prefer GitHub teams over individuals to make it easier to find a suitable owner
 # when someone is OOO or leaves the team.
 
-# The python-samples-owners team is the default owner for anything not
-# explicitly taken by someone else.
-*                               @GoogleCloudPlatform/python-samples-owners
+# python-samples-owners is in charge of infrastructure (Kokoro, noxfiles, etc.) in this repository
```

Comment:
Do we want to add these files for the python-samples-owners?

Reply:
@busunkim96 Would you mind updating this?

Reply:
Specified `github`, `kokoro` and top level directory files to be explicitly owned by python-samples-owners group.